components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -210,13 +212,19 @@
   #set($selectedValues = [])
 #end
 
-{{html clean="false" filter="indent"}}&lt;form class="xform" action="${escapetool.xml($request.get('export_endpoint'))}" method="post"&gt;&lt;div&gt;{{/html}}
+#set ($form_action = $!{escapetool.xml($!{request.get('export_endpoint')})})
+#if (!$form_action)
+  #set ($form_action = "none")
```

Comment:
This will actually send the form to `http://localhost:8080/none`, probably not what we want.